cyipopt/tests/unit/test_scipy_ipopt_from_scipy.py

```diff
@@ -99,70 +100,70 @@ def fprime_ieqcon2(self, x):
 
     # minimize
     def test_minimize_unbounded_approximated(self):
-        # Minimize, method='SLSQP': unbounded, approximated jacobian.
-        jacs = [None, False, '2-point', '3-point']
+        # Minimize, method=None: unbounded, approximated jacobian.
+        jacs = [None, False]
```

Comment:
I suppose if we want the interface to exactly match `minimize`, we need to accept these strings.